Automattic/woocommerce-payments
# Stripe Link: Pre-fill first and last names on checkout

Fixes #4512 

#### Changes proposed in this Pull Request

This PR updates the Link functionality, in order to pre-fill first and last name fields on the checkout form, along with other user details.









#### Testing instructions





* Navigate to Settings -> Payment Methods and enable Link payment method.
* create an order, go to checkout page and complete all checkout fields. Select also 1click payment under credit card fields
* finish the order
* create another order, go to checkout page and fill the email
* a pop-up should be shown to type the pin ( 0 0 0 0 0 0 for tests )
* first and last name should be filled for both billing and shipping address


-------------------


**Post merge**
